tools/pythonpkg/duckdb-stubs/__init__.pyi

```diff
@@ -37,32 +37,30 @@ class DataError(Error): ...
 
 class DuckDBPyConnection:
     def __init__(self, *args, **kwargs) -> None: ...
-    def append(self, table_name: str, df: object) -> DuckDBPyConnection: ...
+    def append(self, table_name: str, df: pandas.DataFrame) -> DuckDBPyConnection: ...
     def arrow(self, chunk_size: int = ...) -> object: ...
     def begin(self) -> DuckDBPyConnection: ...
     def close(self) -> None: ...
     def commit(self) -> DuckDBPyConnection: ...
     def cursor(self) -> DuckDBPyConnection: ...
     @overload
-    def df(self) -> object: ...
+    def df(self) -> pandas.DataFrame: ...
     @overload
-    def df(self, df: object) -> DuckDBPyRelation: ...
-    @overload
-    def df(aliasoffrom_df) -> Any: ...
```

Comment:
This function has been removed now, as it never actually existed - the mypy stub generator was confused by the braces in the docstring

Reply:
(Sligthly unrelated to the PR, but triggered by this comment) Looking at these functions that are a lot of operators (e.g., limit) specialized for DFs, I guess it comes from a time before we had a RAPI, I wonder if it makes sense to keep these. I guess ideally users would need to transform dfs to duckdb rel, use our internal operators, and then output the result to whatever format they want.